core/systemcontracts/upgrade.go

```diff
@@ -364,18 +364,34 @@ func init() {
 			},
 		},
 	}
```

Comment:
For testnet hardfork, what is done is done, we can not change the byte code again, otherwise testnet can not re-sync from genesis.